lib/src/widgets/nes_icon_button.dart

```diff
@@ -49,8 +53,9 @@ class NesIconButton extends StatelessWidget {
       onPress: onPress,
       onPressStart: onPressStart,
       onPressEnd: onPressEnd,
+      disabled: disabled ?? false || _isDisabled(),
       child: Opacity(
-        opacity: _isDisabled() ? .2 : 1.0,
+        opacity: (disabled ?? false || _isDisabled()) ? .2 : 1.0,
```

Comment:
Oh, great catch, maybe instead of doing `disable ?? false` in more than one place, we could simply add it to the conditions on the `_isDisabled` method?